app/views/portal/show.rb

```diff
@@ -420,6 +421,30 @@ def labels
 
     private
 
+    def document_hierarchy
+      {
+        parent: hierarchy_node(@document.hierarchy[:parent]),
+        siblings: {
+          items: @document.hierarchy[:preceding_siblings].map { |item| hierarchy_node(item) } +
+                   [ hierarchy_node(@document.hierarchy[:self]) ] +
```

Comment:
Use 2 (not 9) spaces for indenting an expression spanning multiple lines.<br>Space inside square brackets detected.